lib/dead_end/banner.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module DeadEnd
+  class Banner
+    CHARACTERS = { :"}" => :"{", :"]" => :"[", :")" => :"(" }
+    ALL_CHARACTERS = CHARACTERS.keys + CHARACTERS.values
```

Comment:
Should we just list all combinations in one hash, or does this make sense as is? 

Also, should we freeze the `CHARACTERS` hash?